nixos/modules/security/acme/default.nix

```diff
@@ -566,6 +572,22 @@ let
         example = "/var/src/secrets/example.org-route53-api-token";
       };
 
+      credentialFiles = mkOption {
```

Comment:
This option name is way too similarly named to credentialsFile. I think the description needs to clear up what the difference is first - that this allows you to securely pass any secret file paths which lego must read at runtime by adding them to Systemd's LoadCredential, which will in turn set the permissions appropriately within its mount namespace.
Maybe `secretFiles` would work, as then a user could correlate that with the lego documentation when figuring out how to configure the module.